alacritty_terminal/src/term/search.rs

```diff
@@ -629,80 +665,91 @@ mod tests {
         ");
 
         // Make sure the cell containing the linebreak is not skipped.
-        let dfas = RegexSearch::new("te.*123").unwrap();
+        let regex = RegexSearch::new("te.*123").unwrap();
         let start = Point::new(Line(0), Column(2));
         let end = Point::new(Line(1), Column(9));
         let match_start = Point::new(Line(1), Column(0));
-        assert_eq!(term.regex_search_right(&dfas, start, end), Some(match_start..=end));
+        assert_eq!(term.regex_search_right(&regex, start, end), Some(match_start..=end));
     }
 
     #[test]
     fn skip_dead_cell() {
         let term = mock_term("alacritty");
 
         // Make sure dead state cell is skipped when reversing.
-        let dfas = RegexSearch::new("alacrit").unwrap();
+        let regex = RegexSearch::new("alacrit").unwrap();
         let start = Point::new(Line(0), Column(0));
         let end = Point::new(Line(0), Column(6));
-        assert_eq!(term.regex_search_right(&dfas, start, end), Some(start..=end));
+        assert_eq!(term.regex_search_right(&regex, start, end), Some(start..=end));
     }
 
     #[test]
     fn reverse_search_dead_recovery() {
         let term = mock_term("zooo lense");
 
         // Make sure the reverse DFA operates the same as a forward DFA.
-        let dfas = RegexSearch::new("zoo").unwrap();
+        let regex = RegexSearch::new("zoo").unwrap();
         let start = Point::new(Line(0), Column(9));
         let end = Point::new(Line(0), Column(0));
         let match_start = Point::new(Line(0), Column(0));
         let match_end = Point::new(Line(0), Column(2));
-        assert_eq!(term.regex_search_left(&dfas, start, end), Some(match_start..=match_end));
+        assert_eq!(term.regex_search_left(&regex, start, end), Some(match_start..=match_end));
     }
 
     #[test]
     fn multibyte_unicode() {
         let term = mock_term("testвосибing");
 
-        let dfas = RegexSearch::new("te.*ing").unwrap();
+        let regex = RegexSearch::new("te.*ing").unwrap();
         let start = Point::new(Line(0), Column(0));
         let end = Point::new(Line(0), Column(11));
-        assert_eq!(term.regex_search_right(&dfas, start, end), Some(start..=end));
+        assert_eq!(term.regex_search_right(&regex, start, end), Some(start..=end));
 
-        let dfas = RegexSearch::new("te.*ing").unwrap();
+        let regex = RegexSearch::new("te.*ing").unwrap();
         let start = Point::new(Line(0), Column(11));
         let end = Point::new(Line(0), Column(0));
-        assert_eq!(term.regex_search_left(&dfas, start, end), Some(end..=start));
+        assert_eq!(term.regex_search_left(&regex, start, end), Some(end..=start));
+    }
+
+    #[test]
+    fn end_on_multibyte_unicode() {
+        let term = mock_term("testвосиб");
```

Comment:
That string looks suspicious, but whatever.

Reply:
Suspicious how? I just copied it from a different test.

Reply:
Huh, ok then, it's just a suffix of my surname.